src/pages/view-test-result/cat-adi-part2/components/test-details/test-details.cat-adi-part2.scss

```diff
@@ -0,0 +1,7 @@
+test-details-card-adi2 {
+  .exclamation-indicator {
+    width: 26px;
+    height: 26px;
+    float: right;
+  }
+}
```

Comment:
was including this intentional?

Reply:
yes it was

Reply:
alright - happy to approve then, just seemed out of place having adi2 related stuff in here